### ISSUE TITLE
Ambiguous references on the sisbase.Utils namespace

### ISSUE THREAD

**Issue by alikindsys** [RORIdev/sisbase#31]
Workaround
---
Explicitly declare `DSharpPlus.Permissions`

Proposed Solution
---
Rename `sisbase.Utils.Permissions` to avoid the conflict.

**Comment by wffirilat** [comment]
Suggested name: `sisbase.Utils.PermissionUtils`. `Permissions` is a static class with helper methods, and these can conventionally be postfixed with `Utils`

**Comment by alikindsys** [comment]
Then its better to find all the classes that match that condition and apply the rename. Pretty sure theres more then just one.

**Comment by wffirilat** [comment]
Agreed. The only other file in `Utils` that matches this pattern is `Unicode`, which contains a list of unicode emojis. It does not (currently) provide any functions abstractig over the usage of this list, only two dictionaries `Emojis: Disctionary<string, string>` (really `<UnicodePoint, DiscordName>`) and UnicodeEmojisByDiscordName (really `Dictionary<DiscordName, UnicodePoint>`). these aren't self explanatory, so perhaps split that rename and refactoring into a different issue
